keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt

```diff
@@ -0,0 +1,36 @@
+package com.netflix.spinnaker.keel.services
+
+import com.netflix.spinnaker.igor.BuildService
+import com.netflix.spinnaker.keel.api.artifacts.ArtifactMetadata
+import com.netflix.spinnaker.keel.api.artifacts.BuildMetadata
+import com.netflix.spinnaker.keel.api.artifacts.GitMetadata
+import kotlinx.coroutines.runBlocking
+import org.springframework.stereotype.Component
+
+/**
+ * Provides functionality to convert build metadata, which is coming from internal service, to artfiact metadata (via igor).
+ */
+@Component
+class ArtifactMetadataService(
+  private val buildService: BuildService
+) {
+
+  fun getArtifactMetadata(
```

Comment:
Should also be `suspend`, I think. Then you don't have to call `runBlocking` below.

Reply:
done!